recipes/rgi/meta.yaml

```diff
@@ -38,10 +38,15 @@ requirements:
     - pyfaidx >=0.5.4.1
     - pyahocorasick >=1.1.7
     - setuptools
-
+    - bowtie2 >=2.3.4.3
```

Comment:
I think these dependencies should likely be added to the `run` section instead of the `host` section. The `host` section defines dependencies used during building of the package, while `run` is the run-time requirements. See for example https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run.

Reply:
@apetkau moved dependencies to `run` section